packages/jest-snapshot/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "jest-snapshot",
+  "version": "12.0.2",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/jest.git"
+  },
+  "license": "BSD-3-Clause",
+  "main": "src/index.js",
+  "dependencies": {
+    "jest-util": "^12.0.2",
+    "react": "^15.0.2",
```

Comment:
I see you are using it in the test. We should probably make it a test dependency.